examples/ibm-resource-instance/README.md

```diff
@@ -53,29 +47,32 @@ data "ibm_resource_instance" "test" {
 
 ## Examples
 
-* [ Resource Istance  ](https://github.com/IBM-Cloud/terraform-provider-ibm/tree/master/examples/ibm-resource-instance)
+* [Resource Instance](https://github.com/IBM-Cloud/terraform-provider-ibm/tree/master/examples/ibm-resource-instance)
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
 ## Requirements
 
+You need Terraform v1.0.0 installer to execute this example in your account. For more information, about Terraform installation, see [Installing the Terraform CLI](https://cloud.ibm.com/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-getting-started)
+
 | Name | Version |
 |------|---------|
-| terraform | ~> 0.12 |
+| terraform |>= 1.0.0, < 2.0.0 |
 
 ## Providers
 
 | Name | Version |
 |------|---------|
-| ibm | n/a |
+| ibm | latest |
 
 ## Inputs
 
 | Name | Description | Type | Required |
 |------|-------------|------|---------|
-| name | Name of the resource group. | `string` | yes |
 | service_name | The name of the service offering. | `string` | yes |
 | service_type | The type of the service offering. | `string` | yes |
 | plan| The name of the plan type supported by service.| `string` | yes |
 | location | Target location or environment to create the resource instance. | `string` | yes |
-
+| resource_group | Name of your resource group. | `string` | yes |
+{: caption="Resource input variables" caption-side="top"}
```

Comment:
{: caption="Resource input variables" caption-side="top"} is not rendered proeprly

Reply:
deleted the caption in the table.